crates/spfs-vfs/src/fuse.rs

```diff
@@ -790,6 +807,14 @@ impl fuser::Filesystem for Session {
     }
 
     fn lookup(&mut self, _req: &Request<'_>, parent: u64, name: &OsStr, reply: ReplyEntry) {
+        if name.as_bytes().starts_with(b".fuse-heartbeat") {
+            let seconds_since_session_start = self.inner.session_start.elapsed().as_secs();
+            tracing::trace!(?seconds_since_session_start, "heard heartbeat");
+            self.inner
+                .last_heartbeat_seconds_since_session_start
+                .store(seconds_since_session_start, Ordering::Relaxed);
+        }
```

Comment:
Note that this behavior is a side-effect and the lookup is still processed normally. This could be considered "expensive" overhead that has to communicate with remote spfs repos to process.

The magic filename pattern could include a hard-coded uuid to make it "impossible" that /spfs environment would actually contain this file, and then this code could synthesize the reply itself without processing the lookup normally.

Reply:
I created a "magic" filename prefix.